### PR TITLE
Adds a 'Features or Services' block pattern

### PR DESCRIPTION
**Pull request by enriquesanchez**
Adds a 'Features or Services' block pattern for #20345.

Screen Shot 2020-03-13 at 16 15 06

### CODE REVIEW

**Comment by mtias** [comment]
How would this one look if we leave just placeholders for the images?

**Comment by enriquesanchez** [comment]
@mtias do you mean something like this?

Frame 112

**Comment by enriquesanchez** [comment]
Hi @mtias 👋

Wondering if you have any comments about this pattern. Let me know if you think it can be improved or if we should close this PR instead.

**Comment by mapk** [comment]
It would be great to get Matias' feedback on this, but if the placeholder images aren't favorable, than maybe try gradient blocks? @enriquesanchez a cover block with a gradient might work well there.

**Comment by mtias** [comment]
I mean leaving the image block empty so it just display the placeholder (and you can drag an image into it).

**Comment by enriquesanchez** [comment]
>I mean leaving the image block empty so it just displays the placeholder (and you can drag an image into it).

@mtias @mapk 

Got it, thanks for the clarification. Here's how that would look:

### In the editor
Screen Shot 2020-04-14 at 16 55 49


### On the front end
Screen Shot 2020-04-14 at 16 55 35

**Comment by enriquesanchez** [comment]
We could also use a gradient background for the group blocks containing the image block placeholders:

Screen Shot 2020-04-14 at 17 07 15

**Comment by mapk** [comment]
The gradient one works well. Let's merge it.

**Comment by mtias** [comment]
Let's go with the gradient.

**Review by mtias — APPROVED** [review]

**Comment by enriquesanchez** [comment]
@mtias @mapk One thing we didn't consider is the pattern preview:

Screen Shot 2020-05-04 at 16 19 33


The image placeholders do show up in the preview.

**Comment by enriquesanchez** [comment]
Made some tweaks to the copy to make the preview more balanced (thanks for the suggestion @mapk!)

Screen Shot 2020-05-05 at 11 05 21

File: lib/client-assets.php [hunk 1]
```diff
@@ -691,4 +691,5 @@ function gutenberg_extend_settings_custom_units( $settings ) {
 	register_pattern( 'core/hero-two-columns', gutenberg_load_block_pattern( 'hero-two-columns' ) );
 	register_pattern( 'core/numbered-features', gutenberg_load_block_pattern( 'numbered-features' ) );
 	register_pattern( 'core/its-time', gutenberg_load_block_pattern( 'its-time' ) );
+	register_pattern( 'core/its-time', gutenberg_load_block_pattern( 'features-services' ) );
```
Comment (mtias) [inline lib/client-assets.php:694]: The slug should be updated

Comment (enriquesanchez) [reply lib/client-assets.php:694]: Oh, totally missed it. Thank you!

**Comment by enriquesanchez** [comment]
@mapk @mtias This one is ready to merge, but I still like to hear your opinion on how the imahge placeholders look in the preview.

The too narrow issue can be fixed now that https://github.com/WordPress/gutenberg/pull/22216 is merged, but how do you feel about the placeholder showing up on the preview?

**Comment by mapk** [comment]
I'm not a fan of the Image placeholder. I'd rather we drop that.

**Comment by enriquesanchez** [comment]
@mapk Removed image placeholders and added `viewportWidth`

Preview:

Screen Shot 2020-05-15 at 14 24 36


Editor:

Screen Shot 2020-05-15 at 14 19 32


Front-end:

Screen Shot 2020-05-15 at 14 19 46

**Pull request merged**